examples/chip-tool/BUILD.gn

```diff
@@ -91,10 +91,30 @@ static_library("chip-tool-utils") {
   output_dir = root_out_dir
 }
 
+action("generate_test_paa_trust_store_list") {
+  script = "${chip_root}/credentials/test/gen-test-paa-trust-store-list.py"
+  sources = [ "${chip_root}/credentials/test/attestation" ]
+  outputs = [ "${root_out_dir}/test_trust_store.paa" ]
+
+  args = rebase_path(sources, "") + [ "Chip-Test-PAA", "Cert.pem" ] + rebase_path(outputs, root_out_dir)
+}
+
+action("generate_development_paa_trust_store_list") {
```

Comment:
Suggest making this step manual, with documentation, rather than regen every time, especially since it relies on a network resource. None of the build items in any .gn file can rely directly on network resources